Common/include/CConfig.hpp

```diff
@@ -1184,7 +1186,6 @@ class CConfig {
   bool Species_StrongBC;           /*!< \brief Boolean whether strong BC's are used for in- outlet of the species solver. */
   su2double* Species_Init;         /*!< \brief Initial uniform value for scalar transport. */
   unsigned short nSpecies_Init;    /*!< \brief Number of entries of SPECIES_INIT */
-  unsigned short n_species;        /*!< \brief Number of species equations! Not species itself which would be 1 more. */
```

Comment:
n_species was introduced by @TobiKattmann for the species solver, but we actually use nSpecies already defined by the nemo variables. That's why it was decided to remove n_species and continue using nSpecies.